scripts/20191212_dangling-key-cleanup/delete-keys.js

```diff
@@ -245,7 +245,7 @@ const runScriptsOn = async (Collection, keysToDelete, collectionName) => {
 
   // Define keys to be deleted
   const agencyKeysToDelete = {
-    CorpPass: 1,
+    Corppass: 1,
```

Comment:
might want to revert this, this script is kept here for historical purposes and should not be editted